app/src/main/java/com/sopt/now/compose/presentation/ui/auth/screen/SignUpScreen.kt

```diff
@@ -0,0 +1,215 @@
+            modifier = Modifier
+                .wrapContentSize()
+                .align(Alignment.Start),
+            style = CustomTheme.typography.head2
+        )
+        Spacer(modifier = Modifier.height(18.dp))
+        AuthTextField(
+            value = inputNickname,
+            onValueChange = { inputNickname = it },
+            modifier = Modifier
+                .fillMaxWidth(),
+            isFocused = isNicknameTextFieldFocused,
+            onFocusChanged = { isNicknameTextFieldFocused = it },
+            onRemove = { inputNickname = TextFieldValue("") },
+            hint = stringResource(R.string.signup_nickname_hint)
+        )
+
+        Spacer(modifier = Modifier.height(30.dp))
+
+        Text(
+            text = stringResource(R.string.signup_phone_number_title),
+            modifier = Modifier
+                .wrapContentSize()
+                .align(Alignment.Start),
+            style = CustomTheme.typography.head2
+        )
+        Spacer(modifier = Modifier.height(18.dp))
+        AuthTextField(
+            value = inputPhoneNumber,
+            onValueChange = { inputPhoneNumber = it },
+            modifier = Modifier
+                .fillMaxWidth(),
+            isFocused = isPhoneNumberTextFieldFocused,
+            onFocusChanged = { isPhoneNumberTextFieldFocused = it },
+            onRemove = { inputPhoneNumber = TextFieldValue("") },
+            hint = stringResource(R.string.signup_phone_number_hint)
+        )
+
+        Spacer(modifier = Modifier.weight(1f))
+
+        Button(
+            onClick = {
+                val user = setUser(
+                    inputId.text,
+                    inputPassword.text,
+                    inputPassword.text,
+                    inputPhoneNumber.text
+                )
+                if (SignUpValidation.isSignUpValid(user)) {
+                    showToast(context, context.getString(R.string.signup_signup_success))
+                    onClickSignUp(
+                        inputId.text,
+                        inputPassword.text,
+                        inputNickname.text,
+                        inputPhoneNumber.text
+                    )
+                } else showToast(context, context.getString(R.string.signup_signup_failure))
+            },
+            modifier = Modifier
+                .fillMaxWidth(),
+            colors = ButtonDefaults.buttonColors(CustomTheme.colors.mainYellow),
+            shape = RoundedCornerShape(10.dp)
+        ) {
+            Text(
+                text = stringResource(R.string.signup_signup_button),
+                color = CustomTheme.colors.gray01
+            )
+        }
+    }
+}
+
+fun setUser(
+    inputId: String,
+    inputPassword: String,
+    inputNickname: String,
+    inputPhoneNumber: String
+): User {
+    return User(
+        id = inputId,
+        password = inputPassword,
+        nickname = inputNickname,
+        phoneNumber = inputPhoneNumber
+    )
+}
```

Comment:
```suggestion
fun setUser(
    inputId: String,
    inputPassword: String,
    inputNickname: String,
    inputPhoneNumber: String
): User = User(
    id = inputId,
    password = inputPassword,
    nickname = inputNickname,
    phoneNumber = inputPhoneNumber
)
```

저는 이게 더 코틀린스러워서 좋아해요 ㅎ.ㅎ
근데 취향차이라 지금 방식을 더 선호하시는 분들도 있으셔서 참고만 하세용~

Reply:
확실히 함수형답긴하네요..! 코인액을 열심히 더 파봐야겠습니다 감사합니다 ㅎ